docker/neumann/Dockerfile

```diff
@@ -33,6 +37,12 @@ USER $USERNAME
 
 # Copy files
 COPY --chown=$USERNAME --from=builder /build/target/release/$BINARY  /$USERNAME/$BINARY
+COPY --chown=$USERNAME --from=builder /build/target/release/wbuild/neumann-runtime/neumann_runtime.compact.compressed.wasm  /$USERNAME/neumann_runtime.compact.compressed.wasm 
+COPY --chown=$USERNAME --from=builder /build/target/release/wbuild/neumann-runtime/neumann_runtime.compact.wasm  /$USERNAME/neumann_runtime.compact.wasm
```

Comment:
I don't think these need to be in the production image.
Something like this should let you keep the files in the builder image but still copy them out, I think.
```
docker build --target builder -f ./docker/neumann/Dockerfile -t neumann-builder:latest
docker create --name artifact_holder neumann-builder:latest
docker cp artifact_holder:<file path> ./neumann_runtime.compact.compressed.wasm
docker cp artifact_holder:<file_path> ./neumann_runtime.compact.wasm
```